packages/frontend-shared/src/gql-components/promo/PromoAction.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <component
+    :is="wrapperComponent"
+    data-cy="promo-action"
+    class="flex flex-row items-center gap-[12px] h-[32px] bg-gray-50 border border-gray-100 rounded w-fit text-sm"
+    :class="wrapperClasses"
+    v-bind="wrapperProps"
+    @click="action"
```

Comment:
Argh, was going back and forth on which made more sense - guessed wrong. I think it will be a bit of a struggle to get the border rounding to work correctly but I'll figure something out

Reply:
Reworked to use the DS Button and restrict actionability to the right side. Not sure on the styling - per Figma we're using the `outline-light` variant, but the hocus styles for that are pretty subdued in this context. Will take a closer look once the tests settle down on the latest commits